run_dir/static/js/project_samples.js

```diff
@@ -1200,3 +1210,15 @@ $('#downloadImgsBtn').click(function(e){
   var download_api_url = '/api/v1/download_images/'+project+'/'+option;
   $('#chooseImgType').attr('action', download_api_url).attr('method', 'post');
 });
+
+function load_last_modified(){
```

Comment:
Do we need another function and API for this? The API call for`/api/v1/project_summary/P*` in `load_all_udfs()` already returns `modified`. Wouldn't it be easier to add this in `load_all_udfs()`?

Reply:
That's a good idea! I am looking at it now :-)